homeassistant/components/leneda/diagnostics.py

```diff
@@ -0,0 +1,19 @@
+"""Diagnostics support for Leneda."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+
+async def async_get_config_entry_diagnostics(
```

Comment:
You should strip out any PII / access related data here if not done so already, see the warning here:

https://developers.home-assistant.io/docs/core/integration_diagnostics/

There could be a risk of users uploading the diagnostics into an issue without checking first that their private information is stripped out.

Reply:
Thanks a lot for the hint. I read the documentation and made sure to strip or anonymise potentially sensitive data.